models/aether_cube_sphere/work/demo_documentation.txt

```diff
@@ -0,0 +1,69 @@
+This file provides step-by-step instructions for testing the Aether/DART
+system with a variety of test restart files available from the University of
+Michigan.
+
+Check out aether_michigan branch of dart
+	git clone https://github.com/NCAR/DART.git
+	git checkout aether_michigan
```

Comment:
Not sure if this demo_documentation is supposed to be in the tar file of test data vs. in the aether work directory.
```suggestion
```